schema/deploy/tables/report_type.sql

```diff
@@ -0,0 +1,40 @@
+-- Deploy cif:tables/report_type to pg
+
+
+begin;
+
+create table cif.report_type
+(
+  id integer primary key generated always as identity,
+  name varchar(1000) not null,
+  form_schema jsonb
```

Comment:
```suggestion
```
let's not put this in here for now. Adding the form schemas to the database needs to be done in a systematic manner (see #373).  We'll probably want a `form_schema` table. This is out of scope of this PR.